Serverless/Azure Functions/About Azure Functions.md

```diff
@@ -1,36 +1,57 @@
-Azure Functions is a Function-as-a-Service ("FaaS"), i.e. a platform for running configurable blocks of code that run in response to some event. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
+[Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) is Microsoft's serverless solution. They are, unsurprisingly, hosted on Azure, Microsoft's cloud service. Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
 
-The main benefits of using Azure Functions and serverless solutions are:
-
-* Writing less code - Allows the developer to focus on the business logic implementation and utilize the ready-to-use infrastructure features.
-* Maintain less infrastructure - Cloud infrastructure providers manage servers, security, and scaling. Functions are scaled automatically.
-* Save on costs - No need to buy resources in advance - pay as you go.
+Azure Functions are interlocked with Azure Storage services, meaning that every Azure function needs Azure storage configured to be able to run. The storage account connection is used by the Functions host for operations such as managing triggers and logging function executions. It's also used when dynamically scaling function apps.
 
 ### Scaling
 
-There are three basic hosting plans available for Azure Functions:
+Our scaling options will depend on the hosting plan we have for our service. There are three basic hosting plans available for Azure Functions:
 
-* Consumption plan - Functions scale automatically and billing is calculated only for compute resources used i.e. when functions are running. This means that the functions have a cold start. Cold start implies startup latency, so the first trigger execution after the function was idle takes a longer time.
+* Consumption plan - Functions scale automatically and billing is calculated only for the used resources. This means that the functions have a cold start (you can read more about its implications in our [Serverless computing chapter](../serverless-computing)). A single-function app in this plan scales out to a maximum of 200 instances, while a single instance can process more than one message/request at a time, except for time-triggered functions which are always singletons.
 * Premium plan - Automatically scale using pre-warmed workers which run applications with no delay after being idle. Functions run on more powerful instances and connect to virtual networks. No cold start.
 * Dedicated (App Service) plan. - Run functions within an App Service plan at regular App Service plan rates. This plan is suitable for long-running scenarios such as Durable Functions, or where predictable costs are needed.
```

Comment:
That's something from before, but can you please remove the dot after the `Dedicated (App Service) plan` ?